resources/analysisTools/copyNumberEstimationWorkflow/createUnphasedFiles.sh

```diff
@@ -12,7 +12,7 @@ zcat $dbSNP_FILE | perl $TOOL_ANNOTATE_CNV_VCF  \
 		--columnName genotype \
 		--aColNameLineStart "#CHROM" | \
 		grep -v "^#"  | \
-		$PERL_BINARY $TOOL_PARSE_VCF ${imputeOutputDirectory} $unphasedGenotypesFilePrefix $unphasedGenotypesFileSuffix
+		$PERL_BINARY $TOOL_PARSE_VCF ${phasingOutputDirectory} $unphasedGenotypesFilePrefix $unphasedGenotypesFileSuffix
```

Comment:
The original code already sucks. Try using quotes in Bash by default, and only omit them if necessary. Here

```bash
$PERL_BINARY "$TOOL_PARSE_VCF" "$phasingOutputDirectory" "$unphasedGenotypesFilePrefix" "$unphasedGenotypesFileSuffix"
```

Here, `$PERL_BINARY` is unquoted, because it may be a command, such as `perl -w`. The unnecessary (here) braces are all removed and the variables all quoted, just in case there are spaces in the values, or even more frequently, in case the value is simply empty! Thus with an empty parameter "" there will more likely be a meaningful error message shown by the Perl script, e.g.

`Empty phasing output directory`  rather than `Missing unphased genotypes file`, if `phasingOutputDirectory` is empty.